capa/features/extractors/binexport2/file.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import io
+import logging
+from typing import Tuple, Iterator
+
+import pefile
+from elftools.elf.elffile import ELFFile
+
+import capa.features.common
+import capa.features.extractors.common
+import capa.features.extractors.pefile
+import capa.features.extractors.elffile
+from capa.features.common import Feature
+from capa.features.address import Address
+from capa.features.extractors.binexport2.binexport2_pb2 import BinExport2
+
+logger = logging.getLogger(__name__)
+
+
+def extract_file_export_names(_be2: BinExport2, buf: bytes) -> Iterator[Tuple[Feature, Address]]:
+    if buf.startswith(capa.features.extractors.common.MATCH_PE):
+        pe = pefile.PE(data=buf)
+        yield from capa.features.extractors.pefile.extract_file_export_names(pe)
+    elif buf.startswith(capa.features.extractors.common.MATCH_ELF):
+        elf = ELFFile(io.BytesIO(buf))
+        yield from capa.features.extractors.elffile.extract_file_export_names(elf)
```

Comment:
for `a881dd981034d225cc2a298358fccccc9792df478c81d7e678d12b6658fe266a` ([a881dd981034d225cc2a298358fccccc9792df478c81d7e678d12b6658fe266a.BinExport.txt](https://github.com/mandiant/capa/files/14223564/a881dd981034d225cc2a298358fccccc9792df478c81d7e678d12b6658fe266a.BinExport.txt)) exports and imports don't seem to work

Reply:
This appears unrelated to the BinExport code and instead an issue with how we're extracting import / export symbols from ELF files in general. I'm going to close this comment here in favor of [the new issue that I've created to track this](https://github.com/mandiant/capa/issues/2096).